src/main/java/com/jcabi/beanstalk/maven/plugin/WarFile.java

```diff
@@ -72,6 +72,7 @@ public WarFile(final ZipFile file) {
      * @throws org.apache.maven.plugin.MojoFailureException Thrown, if the
      *  .ebextensions does not exist in the WAR file, is empty or one of its
      *  files is neither valid JSON, nor valid YAML.
+     *  @todo: move this logic to AbstractBeanstalkMojo
```

Comment:
@tmjee the puzzle description should contains at least 20 words otherwise `Rultor` will reject it